### ISSUE TITLE
Orphaned udevadm process

### ISSUE THREAD

**Issue by meteorfox** [cockpit-project/cockpit#13348]
Cockpit version: 202.1
OS: Ubuntu 19.10
Page: Docker Containers

Each time I open or refresh the Docker Containers page, an udevadm process is forked (which seems to be forked by cockpit-bridge), later the process is orphaned and `ps` shows multiple of them.

Steps to reproduce

1. Make sure to have Docker installed
2. Visit Docker Containers pages under Host tab
3. Check `ps` output on the machine

`ps` output excerpt:

```
...
root      3216  0.0  0.0   7596  2064 ?        S    01:42   0:00 udevadm monitor -u -s block
root      3362  0.0  0.0   7596  1996 ?        S    01:42   0:00 udevadm monitor -u -s block
cockpit+ 25254  0.0  0.0  12372  8528 ?        Ss   18:01   0:02 /usr/lib/cockpit/cockpit-tls
cockpit+ 25264  0.1  0.0 159656  7488 ?        Sl   18:01   0:05  \_ /usr/lib/cockpit/cockpit-ws --for-tls-proxy --port 0
root     25266  0.0  0.0   6792  4184 ?        S    18:01   0:00      \_ /usr/lib/cockpit/cockpit-session localhost
meteorf+ 25280  0.1  0.0 315172 11548 ?        Sl   18:01   0:05          \_ cockpit-bridge
root     25289  0.0  0.0 312604 12664 ?        Sl   18:01   0:01              \_ /usr/bin/cockpit-bridge --privileged
root      3549  4.0  0.0  16788 11428 ?        S    18:57   0:00              |   \_ /usr/bin/python3 - monitor
root      3584  0.0  0.0   7596  1932 ?        S    18:57   0:00              |       \_ udevadm monitor -u -s block
meteorf+ 25293  7.7  0.0  96252 14336 ?        Sl   18:01   4:20              \_ /usr/lib/cockpit/cockpit-pcp
meteorf+ 25269  0.0  0.0   5996   472 ?        Ss   18:01   0:00 /usr/bin/ssh-agent
root     25300  0.0  0.0  15984  6108 ?        Ss   18:01   0:00 /lib/systemd/systemd-timedated
root     25390  0.0  0.0   7596  1932 ?        S    18:01   0:00 udevadm monitor -u -s block
root     25727  0.0  0.0   7596  2000 ?        S    18:04   0:00 udevadm monitor -u -s block
root     25803  0.0  0.0   7596  1932 ?        S    18:05   0:00 udevadm monitor -u -s block
root     25854  0.0  0.0   7596  2060 ?        S    18:05   0:00 udevadm monitor -u -s block
root     25950  0.0  0.0   7596  1932 ?        S    18:05   0:00 udevadm monitor -u -s block
root     26026  0.0  0.0   7596  1996 ?        S    18:05   0:00 udevadm monitor -u -s block
root     26113  0.0  0.0   7596  2000 ?        S    18:05   0:00 udevadm monitor -u -s block
root     29253  0.0  0.0   7596  1988 ?        S    18:14   0:00 udevadm monitor -u -s block
root      1651  0.0  0.0   7596  1996 ?        S    18:40   0:00 udevadm monitor -u -s block
root      1727  0.0  0.0   7596  2064 ?        S    18:40   0:00 udevadm monitor -u -s block
root      1799  0.0  0.0   7596  2000 ?        S    18:40   0:00 udevadm monitor -u -s block
root      2213  0.0  0.0   7596  1996 ?        S    18:45   0:00 udevadm monitor -u -s block
root      2328  0.0  0.0   7596  2064 ?        S    18:46   0:00 udevadm monitor -u -s block
root      2429  0.0  0.0   7596  1996 ?        S    18:51   0:00 udevadm monitor -u -s block
root      2526  0.0  0.0   7596  1996 ?        S    18:51   0:00 udevadm monitor -u -s block
root      2772  0.0  0.0   7596  1936 ?        S    18:53   0:00 udevadm monitor -u -s block
root      2867  0.0  0.0   7596  1996 ?        S    18:53   0:00 udevadm monitor -u -s block
root      2964  0.0  0.0   7596  1936 ?        S    18:54   0:00 udevadm monitor -u -s block
root      3476  0.0  0.0   7596  1932 ?        S    18:57   0:00 udevadm monitor -u -s block
root      3524  6.3  0.0  15976  6056 ?        Ss   18:57   0:00 /lib/systemd/systemd-hostnamed
root      3541 25.0  0.1 354916 17256 ?        Ssl  18:57   0:00 /usr/lib/packagekit/packagekitd

```

**Comment by mvollmer** [comment]
I want to look into this as part of #13222.

**Comment by mvollmer** [comment]
(Sorry, this took a bit.)

The `udevadm` process is spawned by the `cockpit-atomic-storage` script, and that script has been removed in #13678, released in Cockpit 214.1.  So your exact issue should have disappeared since 214.1.

However, we also have at least one other instance of that problem in `luksmeta-monitor-hack`, see #13983.

Please check version 214.1 or later.  Thanks for the report!
